packages/devtools-launchpad/src/components/shared/menu.js

```diff
@@ -71,11 +71,13 @@ function showMenu(evt, items) {
   }
 
   let menu = new Menu();
-  items.filter((item) => item.visible).forEach((item) => {
-    let menuItem = new MenuItem(item);
-    menuItem.submenu = createSubMenu(item.submenu);
-    menu.append(menuItem);
-  });
+  items
+    .filter((item) => item.visible === undefined || item.visible === true)
```

Comment:
& it will be nice to provide some unit test to protect the menu not be broken again